desktop/core/src/desktop/js/ko/components/assist/ko.assistPanel.js

```diff
@@ -45,7 +45,7 @@ const TEMPLATE = `
     <!-- ko if: availablePanels().length > 1 -->
     <div class="assist-panel-switches">
       <!-- ko foreach: availablePanels -->
-      <div class="inactive-action assist-type-switch" data-bind="click: function () { $parent.visiblePanel($data); }, css: { 'blue': $parent.visiblePanel() === $data }, style: { 'float': rightAlignIcon ? 'right' : 'left' },  attr: { 'title': name }">
+      <div class="inactive-action assist-type-switch" data-testid="assist--show-sql-panel--button" data-bind="click: function () { $parent.visiblePanel($data); }, css: { 'blue': $parent.visiblePanel() === $data }, style: { 'float': rightAlignIcon ? 'right' : 'left' },  attr: { 'title': name }">
```

Comment:
This also needs a condition (not all panels in availablePanels is a sql panel)

Reply:
Can we generalize it to:
data-testid="assist--show-panel--button"

Reply:
But then it's not useful for when you need to open a specific panel in the tests